app/models/api/submission.rb

```diff
@@ -2,5 +2,8 @@ module API
   class Submission < Base
     has_many :entries, class_name: 'SubmissionEntry'
     has_many :files, class_name: 'SubmissionFile'
+
+    # GET /submissions/:id/calculate
+    custom_endpoint :calculate, on: :member, request_method: :get
```

Comment:
This will need updating - re: https://github.com/dxw/DataSubmissionServiceAPI/pull/9#discussion_r202697742